src/ui/components/Header/UserOptions.tsx

```diff
@@ -53,25 +61,58 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
           expanded={expanded}
           orientation="bottom"
         >
+          {viewMode === "dev" && ( // only show layout options in devtools
+            <>
+              <div className="row group">
+                <span className="flex-1 text-left">Layout</span>
+
+                <div className="flex space-x-1">
+                  <div onClick={() => onLayoutChange("ide")}>
+                    <Icon
+                      filename="dock-bottom-right"
+                      className={`${styles.icon} ${toolboxLayout === "ide" ? styles.selected : ""}`}
+                    />
+                  </div>
+
+                  <div onClick={() => onLayoutChange("left")}>
+                    <Icon
+                      filename="dock-left"
+                      className={`${styles.icon} ${
+                        toolboxLayout === "left" ? styles.selected : ""
+                      }`}
+                    />
+                  </div>
+
+                  <div onClick={() => onLayoutChange("bottom")}>
+                    <Icon
+                      filename="dock-bottom"
+                      className={`${styles.icon} ${
+                        toolboxLayout === "bottom" ? styles.selected : ""
+                      }`}
+                    />
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.divider}></div>
+            </>
+          )}
```

Comment:
I think we also regressed for Node recordings with this change (`recordingCapabilities.supportsRepaintingGraphics`). I totally missed that.